### DISCUSSION TITLE
Programatically add cron jobs

### DISCUSSION THREAD

**Discussion by ivanjeremic** [vercel/community#6322] [Feature Question] [answered]
### Description

I just found out that Vercel offers cron jobs, is there a programmatic API that allows us to add jobs programmatically?

**Comment by amyegan** [comment, marked as answer]
Hey @ivanjeremic! Cron Jobs can be added through [vercel.json](https://vercel.com/docs/projects/project-configuration#crons) or the [Build Output API](https://vercel.com/docs/build-output-api/v3/configuration#crons).

https://vercel.com/docs/cron-jobs

**Reply by ivanjeremic** [reply]
So no programatic api :( very sad. I hope that will be a feature in the future.
